.github/actions/veracode/action.yml

```diff
@@ -4,6 +4,11 @@ inputs:
   srcclr-api-token:
     description: "Agent API Token"
     required: true
+  srcclr-project-ext:
+    description: "Veracode project name extension"
+    required: false
+  # TODO: Add default options + custom options for Source Clear scan, need feedback from Veracode
+  # Below inputs are broken for now
```

Comment:
I understand the 2 previous inputs are broken, but isn't there anything at all that we should implement so that each project - if needed - can pass their own custom options to the srcclr command?

Reply:
There might be - modify the projects srcclr.yml file with the action to move the value from input of action into that location, a bit cumbersome and complicated imo, don't know if it's worth it necessarily.